pkg/urbit/worker/main.c

```diff
@@ -735,6 +753,15 @@ _worker_work_boot(c3_d evt_d, u3_noun job)
     u3V.mug_l  = u3r_mug(u3A->roc);
     u3A->ent_d = u3V.dun_d;
 
+    {
+      u3_noun ver = u3v_wish("zuse");
+      u3m_p("zuse", ver);
```

Comment:
I don't think we need to print these. But if you want to, they should go back over the pipe with `u3l_log()`